backend/app/AppComponents.scala

```diff
@@ -83,6 +85,7 @@ class AppComponents(context: Context, config: Config)
     // data storage services
     val ingestStorage = S3IngestStorage(s3Client, config.s3.buckets.ingestion, config.s3.buckets.deadLetter).valueOr(failure => throw new Exception(failure.msg))
     val blobStorage = S3ObjectStorage(s3Client, config.s3.buckets.collections).valueOr(failure => throw new Exception(failure.msg))
+    val transcriptionStorage = S3ObjectStorage(s3Client, config.s3.buckets.transcription).valueOr(failure => throw new Exception(failure.msg))
```

Comment:
could this be `transcriptStorage` or `transcriptionOutputStorage` - just to make clear that it should only have transcripts in it, not source media